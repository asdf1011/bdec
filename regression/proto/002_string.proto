message String {
 required string value = 2;
}
