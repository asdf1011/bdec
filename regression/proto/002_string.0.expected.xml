<String>
    <value>testing</value>
</String>
